FirebaseAuth/Tests/Sample/SwiftApiTests/SwiftApi.swift

```diff
@@ -0,0 +1,458 @@
+    _ = AuthErrorCode.requiresRecentLogin
+    _ = AuthErrorCode.providerAlreadyLinked
+    _ = AuthErrorCode.noSuchProvider
+    _ = AuthErrorCode.invalidUserToken
+    _ = AuthErrorCode.networkError
+    _ = AuthErrorCode.userTokenExpired
+    _ = AuthErrorCode.invalidAPIKey
+    _ = AuthErrorCode.userMismatch
+    _ = AuthErrorCode.credentialAlreadyInUse
+    _ = AuthErrorCode.weakPassword
+    _ = AuthErrorCode.appNotAuthorized
+    _ = AuthErrorCode.expiredActionCode
+    _ = AuthErrorCode.invalidActionCode
+    _ = AuthErrorCode.invalidMessagePayload
+    _ = AuthErrorCode.invalidSender
+    _ = AuthErrorCode.invalidRecipientEmail
+    _ = AuthErrorCode.missingEmail
+    _ = AuthErrorCode.missingIosBundleID
+    _ = AuthErrorCode.missingAndroidPackageName
+    _ = AuthErrorCode.unauthorizedDomain
+    _ = AuthErrorCode.invalidContinueURI
+    _ = AuthErrorCode.missingContinueURI
+    _ = AuthErrorCode.missingPhoneNumber
+    _ = AuthErrorCode.invalidPhoneNumber
+    _ = AuthErrorCode.missingVerificationCode
+    _ = AuthErrorCode.invalidVerificationCode
+    _ = AuthErrorCode.missingVerificationID
+    _ = AuthErrorCode.invalidVerificationID
+    _ = AuthErrorCode.missingAppCredential
+    _ = AuthErrorCode.invalidAppCredential
+    _ = AuthErrorCode.sessionExpired
+    _ = AuthErrorCode.quotaExceeded
+    _ = AuthErrorCode.missingAppToken
+    _ = AuthErrorCode.notificationNotForwarded
+    _ = AuthErrorCode.appNotVerified
+    _ = AuthErrorCode.captchaCheckFailed
+    _ = AuthErrorCode.webContextAlreadyPresented
+    _ = AuthErrorCode.webContextCancelled
+    _ = AuthErrorCode.appVerificationUserInteractionFailure
+    _ = AuthErrorCode.invalidClientID
+    _ = AuthErrorCode.webNetworkRequestFailed
+    _ = AuthErrorCode.webInternalError
+    _ = AuthErrorCode.webSignInUserInteractionFailure
+    _ = AuthErrorCode.localPlayerNotAuthenticated
+    _ = AuthErrorCode.nullUser
+    _ = AuthErrorCode.dynamicLinkNotActivated
+    _ = AuthErrorCode.invalidProviderID
+    _ = AuthErrorCode.tenantIDMismatch
+    _ = AuthErrorCode.unsupportedTenantOperation
+    _ = AuthErrorCode.invalidDynamicLinkDomain
+    _ = AuthErrorCode.rejectedCredential
+    _ = AuthErrorCode.gameKitNotLinked
+    _ = AuthErrorCode.secondFactorRequired
+    _ = AuthErrorCode.missingMultiFactorSession
+    _ = AuthErrorCode.missingMultiFactorInfo
+    _ = AuthErrorCode.invalidMultiFactorSession
+    _ = AuthErrorCode.multiFactorInfoNotFound
+    _ = AuthErrorCode.adminRestrictedOperation
+    _ = AuthErrorCode.unverifiedEmail
+    _ = AuthErrorCode.secondFactorAlreadyEnrolled
+    _ = AuthErrorCode.maximumSecondFactorCountExceeded
+    _ = AuthErrorCode.unsupportedFirstFactor
+    _ = AuthErrorCode.emailChangeNeedsVerification
+    _ = AuthErrorCode.missingOrInvalidNonce
+    _ = AuthErrorCode.missingClientIdentifier
+    _ = AuthErrorCode.keychainError
+    _ = AuthErrorCode.internalError
+    _ = AuthErrorCode.malformedJWT
```

Comment:
This makes it very obvious to me that we aren't scoping errors well for Swift - ex. `weakPassword` can't be an error on `checkActionCode`. No action needed at the moment but worth evaluating how we can improve this.